src/orb.core.js

```diff
@@ -214,11 +238,13 @@ const orbify = function(X, Y, cb, args = {}) {
     }
 
     async function findPoints() {
+      console.log('find points');
       if (await cornersArray.length) {
         return true;
       }
 
-      window.requestAnimationFrame(findPoints);
+      // repeat this method as fast as the browser can run it
+      if (args.loop) window.requestAnimationFrame(findPoints);
```

Comment:
Hi @rexagod  - hope you're well - the purpose of this iteration wasn't clear to me - why do we repeat `findPoints` and `findMatchedPoints`? When I stop the looping, the matching doesn't work reliably. But when they loop, they don't stop. Shouldn't we end the looping once matches are found? And why do we have to loop in the first place? Thanks!